app/ante/vesting.go

```diff
@@ -48,11 +50,12 @@ func NewEthVestingTransactionDecorator(ak evmtypes.AccountKeeper) EthVestingTran
 // This AnteHandler decorator will fail if:
 //   - the message is not a MsgEthereumTx
 //   - sender account cannot be found
-//   - tx values are in excess of the user's spendable balances
+//   - tx values are in excess of any account's spendable balances
 //   - blocktime is before surpassing vesting cliff end (with zero vested coins)
 func (vtd EthVestingTransactionDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool, next sdk.AnteHandler) (newCtx sdk.Context, err error) {
```

Comment:
Include `AnteHandler` checks, so we can avoid processing tx's that we know are bound to fail